x-pack/plugins/canvas/canvas_plugin_src/renderers/embeddable/embeddable.tsx

```diff
@@ -73,8 +73,7 @@ const renderReactEmbeddable = ({
   );
 };
 
-const renderEmbeddableFactory = (core: CoreStart, plugins: StartDeps) => {
-  const I18nContext = core.i18n.Context;
+const renderEmbeddableFactory = (core: CoreStart, _plugins: StartDeps) => {
```

Comment:
Note: Prepending a parameter with a `_` character silences a warning in the editor about it being an unused variable.